__tests__/app.test.js

```diff
@@ -105,4 +105,55 @@ describe('/api/articles',()=>{
       })
     })
   })
+})
+describe("/api/articles/:article_id/comments",()=>{
+  test("GET:200 should return all comments for an article",()=>{
+    return request(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then((response)=>{
+      const body = response.body
+      expect(body.comments.length).toBe(11)
+      body.comments.forEach((comment)=>{
+        expect(typeof comment.comment_id).toBe('number')
+        expect(typeof comment.votes).toBe('number')
+        expect(typeof comment.created_at).toBe('string')
+        expect(typeof comment.author).toBe('string')
+        expect(typeof comment.body).toBe('string')
+        expect(comment.article_id).toBe(1)
+      })
+    })
+  })
```

Comment:
What if you have an article that has no comments? Ie you could write a test for   **.get('/api/articles/2/comments')** as this has no comments, you would expect a 200 and an empty array